include/hpp/pinocchio/liegroup.hh

```diff
@@ -67,138 +67,218 @@ struct DefaultLieGroupMap {
+    ::pinocchio::JointModelRevoluteUnalignedTpl<Scalar, Options>> {
   typedef liegroup::VectorSpaceOperation<1, true> type;
 };
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelRevoluteUnboundedUnalignedTpl<Scalar, Options> > {
+    ::pinocchio::JointModelRevoluteUnboundedUnalignedTpl<Scalar, Options>> {
   typedef liegroup::SpecialOrthogonalOperation<2> type;
 };
 
 // JointModelPrismaticTpl, JointModelPrismaticUnaligned, JointModelTranslation
 template <typename Scalar, int Options, int Axis>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelPrismaticTpl<Scalar, Options, Axis> > {
+    ::pinocchio::JointModelPrismaticTpl<Scalar, Options, Axis>> {
   typedef liegroup::VectorSpaceOperation<1, false> type;
 };
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelPrismaticUnalignedTpl<Scalar, Options> > {
+    ::pinocchio::JointModelPrismaticUnalignedTpl<Scalar, Options>> {
   typedef liegroup::VectorSpaceOperation<1, false> type;
 };
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelTranslationTpl<Scalar, Options> > {
+    ::pinocchio::JointModelTranslationTpl<Scalar, Options>> {
   typedef liegroup::VectorSpaceOperation<3, false> type;
 };
 
 // JointModelSpherical, JointModelSphericalZYX,
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelSphericalTpl<Scalar, Options> > {
+    ::pinocchio::JointModelSphericalTpl<Scalar, Options>> {
   typedef liegroup::SpecialOrthogonalOperation<3> type;
 };
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelSphericalZYXTpl<Scalar, Options> > {
+    ::pinocchio::JointModelSphericalZYXTpl<Scalar, Options>> {
   typedef liegroup::VectorSpaceOperation<3, true> type;
 };
 
 // JointModelFreeFlyer, JointModelPlanar
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelFreeFlyerTpl<Scalar, Options> > {
+    ::pinocchio::JointModelFreeFlyerTpl<Scalar, Options>> {
   typedef liegroup::SpecialEuclideanOperation<3> type;
 };
 template <typename Scalar, int Options>
 struct DefaultLieGroupMap::operation<
-    ::pinocchio::JointModelPlanarTpl<Scalar, Options> > {
+    ::pinocchio::JointModelPlanarTpl<Scalar, Options>> {
   typedef liegroup::SpecialEuclideanOperation<2> type;
 };
 
+// New in pinocchio 3
+// JointModelMimic
+template <typename OtherJointType>
+struct DefaultLieGroupMap::operation<
+    ::pinocchio::JointModelMimic<OtherJointType>> {
+  typedef typename DefaultLieGroupMap::operation<OtherJointType>::type type;
+};
+
+// JointModelHelical
+template <typename Scalar, int Options, int _axis>
+struct DefaultLieGroupMap::operation<
+    ::pinocchio::JointModelHelicalTpl<Scalar, Options, _axis>> {
+  typedef liegroup::VectorSpaceOperation<1, true> type;
```

Comment:
I guess a helical joint is parameterized by the rotation angle. If not, I think `liegroup::VectorSpaceOperation` will not do exactly what we want.

Reply:
Helical joints are not supported at the moment. This code makes the compilation work.

Reply:
Then can you add a comment for stating this pls?